testv2/tests.yml

```diff
@@ -0,0 +1,102 @@
+- infrastructure: aws_defaults
+  kubernetesVersions:
+    - v1.21.12
+    - v1.22.9
+    - v1.23.6
```

Comment:
How are those Kubernetes versions applied to the upgrade scenarios?

Reply:
Upgrade scenario are using those as "init version" and then uses `params[] .upgradeTo` as upgrade target.